packages/playwright-core/src/utils/zones.ts

```diff
@@ -53,6 +64,20 @@ class ZoneManager {
   }
 }
 
+export class ZoneReference {
```

Comment:
What's is zone reference? I thought zone was a zone reference already. Real zone is the async context. I don't think we need 3 types, should be just async context and a logical apiCall zone with a parent.